utils/src/deployer.rs

```diff
@@ -993,4 +1062,58 @@ mod tests {
         assert_eq!(stake_table.lightClient().call().await?._0, lc_addr);
         Ok(())
     }
+
+    #[tokio::test]
+    async fn test_deploy_timelock() -> Result<()> {
```

Comment:
This function currently isn't hooked up in `deploy.rs`. So it will never be called. Is this intentional?